region/src/main/java/org/motechproject/nms/region/domain/validation/VillageValidator.java

```diff
@@ -15,10 +15,17 @@ public void initialize(ValidVillage validVillage) {
 
     @Override
     public boolean isValid(Village village, ConstraintValidatorContext constraintValidatorContext) {
+        if (village == null) {
+            return true;
+        }
+
         if (village.getVcode() == null && village.getSvid() == null) {
+            constraintValidatorContext
```

Comment:
Except it currently throws an npe.  I put a ticket in platform for it.
On May 13, 2015 7:39 AM, "Frank Huster" <notifications@github.com> wrote:

> In
> region/src/main/java/org/motechproject/nms/region/domain/validation/VillageValidator.java
> <https://github.com/motech-implementations/mim/pull/230#discussion_r30236767>
> :
>
> >          if (village.getVcode() == null && village.getSvid() == null) {
> > +            constraintValidatorContext
>
> Sweet!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/230/files#r30236767>.
>